nyc_taxis/operations/default.py

```diff
@@ -0,0 +1,30 @@
+import random
+
+
+def random_fare(workload, params, **kwargs):
```

Comment:
rally documentation mentioned the parameter `workload` to be `track`, but i repalced it with `workload` after going through os benchmark code. Not sure if I'm doing it right though.

Reply:
Rally's terminology is different; `workload` is the correct term here.